src/api/handlers/userprofile.py

```diff
@@ -267,17 +268,29 @@ def complete_imported_user(self, request):
   def handle_contacts_csv(self, request):
     context: Context = request.context
     args: dict = context.args
-    # find the community within the team that the 
-    csv_ref = args['csv'].file 
-    # csv_ref is a bytes object, we need a csv
-    # so we copy it as a csv temporarily to the disk
-    temporarylocation="testout.csv"
-    with open(temporarylocation, 'wb') as out:
-      var = csv_ref.read()
-      out.write(var)
-    info, err = self.service.handle_csv(context, args, temporarylocation)
+    try:
+    
+      # find the community within the team that the 
+      csv_ref = args['csv'].file 
```

Comment:
nit: a bit unsafe.  
Either asset that csv is inside args or use `args.get('csv')` and maje sure its not None before you  try to get file from it.